.upptimerc.yml

```diff
@@ -122,7 +122,23 @@ sites:
     icon: https://raw.githubusercontent.com/thomasmerz/upptime/master/assets/openvpn_104297.png
 # --- MYPDNS-GITLAB
   - name: MyPDNS Gitlab
-    url: https://mypdns.org
+    url: https://mypdns.org/-/liveness?token=8QTcgMMaEQgzGtdxwPui%27
```

Comment:
@spirillen , "%27" shouldn't belong to URI. Can you please fix this? Thank you.

Reply:
Your right :) will do

PS: 
Have posted so,e thoughts on switching my gitlab to ie YouTrack here https://mypdns.org/mypdns/support/-/issues/800/ comments are welcome